app/src/main/java/org/schabi/newpipe/settings/PeertubeInstanceListFragment.java

```diff
@@ -145,10 +148,13 @@ public void onCreateOptionsMenu(final Menu menu, final MenuInflater inflater) {
         final MenuItem restoreItem = menu
                 .add(Menu.NONE, MENU_ITEM_RESTORE_ID, Menu.NONE, R.string.restore_defaults);
         restoreItem.setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
-
-        final int restoreIcon = ThemeHelper
-                .resolveResourceIdFromAttr(requireContext(), R.attr.ic_restore_defaults);
-        restoreItem.setIcon(AppCompatResources.getDrawable(requireContext(), restoreIcon));
+        final Drawable icon = AppCompatResources.getDrawable(requireContext(),
+                R.drawable.ic_settings_backup_restore);
+        if (icon != null) {
+            icon.mutate();
+            icon.setColorFilter(Color.WHITE, PorterDuff.Mode.SRC_ATOP);
```

Comment:
Shouldn't you add another icon for this case, too? Or am I missing something?

Reply:
SettingsActivity uses AppBar when other components use Toolbar, which accepts `iconTint` theme attribute - it can't be applied to AppBar. I can add a white drawable or delete the black one and replace it with white because it is not used in other places.